source/extensions/omni.isaac.lab/omni/isaac/lab/assets/articulation/articulation.py

```diff
@@ -1348,9 +1348,9 @@ def _process_actuators_cfg(self):
                 self.write_joint_velocity_limit_to_sim(actuator.velocity_limit, joint_ids=actuator.joint_indices)
                 self.write_joint_armature_to_sim(actuator.armature, joint_ids=actuator.joint_indices)
                 self.write_joint_friction_to_sim(actuator.friction, joint_ids=actuator.joint_indices)
-                # Store the actual default stiffness and damping values for explicit actuators (not written the sim)
-                self._data.default_joint_stiffness[:, actuator.joint_indices] = actuator.stiffness
-                self._data.default_joint_damping[:, actuator.joint_indices] = actuator.damping
+            # Store the actual default stiffness and damping values for explicit actuators (not written the sim)
```

Comment:
could you also update the comment here since it also applies to implicit actuators now?

Reply:
I just updated the comment! Thanks for reviewing